docs/content/operations/upgrading/faq.md

```diff
@@ -46,11 +46,17 @@ Review the following changes made to Gloo Gateway in version {{< readfile file="
 
 ### Breaking changes
 
+<!--For 1.20 -->
+
+**Discovery disabled by default in 1.20**: To improve performance, discovery is now disabled by default. This change means that Upstreams are no longer automatically created for discovered Kubernetes services. If you want to continue using discovery, set `discovery.enabled=true` in your Helm values file before upgrading. For more information, see the [Discovery guide]({{< versioned_link_path fromRoot="/installation/advanced_configuration/fds_mode/" >}}).
```

Comment:
I'm wondering if this needs to be explicitly called out in the upgrade guide as an optional in-guide step. not sure how intensely this will break some customers

Reply:
I was even debating whether to put it in breaking changes or not. I don't think it will _break_ anything, so much as they will no longer get new services discovered if they were expecting that.

Reply:
added!